data/beaconrestapi/src/integration-test/java/tech/pegasys/artemis/beaconrestapi/beacon/PostDutiesIntegrationTest.java

```diff
@@ -36,22 +39,40 @@
 
   @Test
   public void shouldReturnNoContentIfStoreNotDefined() throws Exception {
+    final UnsignedLong epoch = UnsignedLong.ONE;
     when(chainStorageClient.getStore()).thenReturn(null);
+    when(chainStorageClient.getFinalizedEpoch()).thenReturn(epoch);
 
-    final Response response = post(1, keys);
+    final Response response = post(epoch.intValue(), keys);
```

Comment:
i wasn't too worried about the literal '1', i was just saying `UnsignedLong.ONE` could easily be static imported and just `ONE`...